frontend/src/components/Overlay/OverlayHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import OverlayCloseIcon from './OverlayCloseIcon';
+
+import './OverlayHeader.css';
+
+interface OverlayHeaderProps {
+	closeOverlay: () => void;
+	heading: string;
+	iconColor?: string;
+}
+
+function OverlayHeader({
+	closeOverlay,
+	heading,
+	iconColor,
+}: OverlayHeaderProps) {
+	const iconFill: string = iconColor ?? '#000';
```

Comment:
```suggestion
	const iconFill: = iconColor ?? '#000';
```

the ts compiler will infer the type from the expression